src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -133,6 +137,23 @@ public void handleLoanUpdate(LoanDTO loan) {
         reportItemInTransit(transaction);
       });
     }
+
+    if (loan.getAction().equals("renewed")) {
+      var folioLoanId = loan.getId();
+
+      transactionRepository.fetchOneByLoanId(folioLoanId).ifPresent(transaction -> {
+        log.info("Loan with id: {} associated with transaction. Transaction id: {}", folioLoanId, transaction.getId());
```

Comment:
The message better describe the specific loan update being handled here, take a look at other handlers

Reply:
Done